common/app/com/commercetools/sunrise/common/ctp/CtpModule.java

```diff
@@ -20,6 +21,7 @@ protected void configure() {
         bind(HttpClient.class).toInstance(SphereClientFactory.of().createHttpClient());
         bind(SphereClientConfig.class).toProvider(SphereClientConfigProvider.class).in(Singleton.class);
         bind(SphereAccessTokenSupplier.class).toProvider(SphereAccessTokenSupplierProvider.class).in(Singleton.class);
+        bind(SphereClient.class).annotatedWith(Names.named("singleton")).toProvider(SphereClientProvider.class).in(Singleton.class);
```

Comment:
@schleichardt after 5 seconds this is the first name that came to my mind, suggestions?